akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -976,6 +976,30 @@ final case class `X-Forwarded-For`(addresses: immutable.Seq[RemoteAddress]) exte
   def getAddresses: Iterable[jm.RemoteAddress] = addresses.asJava
 }
 
+object `X-Forwarded-Host` extends ModeledCompanion[`X-Forwarded-Host`] {
+  implicit val hostRenderer = UriRendering.HostRenderer // cache
+}
+final case class `X-Forwarded-Host`(host: Uri.Host) extends jm.headers.XForwardedHost
```

Comment:
I wonder if we should mark them as `@ApiMayChange` if the specification is not yet completely done, so we reserve the right to change the model later on?

Reply:
Open to that, they are X headers after all...